### ISSUE TITLE
Subsite overview pages don't have a h1 by default

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_subsites#116]
e.g. https://demo.localgovdrupal.org/fostering

Perhaps the title should be marked `h1`?

**Comment by danchamp** [comment]
Possible duplicate of #26?

**Comment by keelanfh** [comment]
Possibly... that refers to subsite pages, whereas this is subsite overviews. The banner mentioned there could be a solution for subsite overviews too, but I don't know enough about this functionality really.

**Comment by bisd** [comment]
It could just be me but none of the Subsites Overviews seems have an h1? Eg:
https://test.localgovdrupal.org/fostering
https://test.localgovdrupal.org//test-subsite-demo-all-components

Withh reference to https://github.com/localgovdrupal/localgov_subsites/issues/26 neither do the Subsites pages as far as I can see?

**Comment by andybroomfield** [comment]
The h1 is provided by the banner paragraph, I suggest some default if a banner paragraph isn't present so the page title renders.

**Comment by bisd** [comment]
Thanks very much @andybroomfield. I am still not clear what a "banner paragraph" is? I am looking at the admin for the following page:
https://test.localgovdrupal.org//test-subsite-demo-all-components

and what I see when I go to the editor for the page is the following:

![banner_text](https://user-images.githubusercontent.com/2633608/231211307-d5422b7c-4fcb-462a-8a20-ff56bdfbf41f.png)

Is the "Text" field under the "Banner and Colour theme" tab what is meant by a "banner paragraph" or is it something else I am not aware of?

Many thanks for your help.

**Comment by msayoung** [comment]
Out of the box the banner titles aren't H1s unfortunately. 
See the demo site: https://demo.localgovdrupal.org/fostering 

We also have two banner types, one has a title field and the other not. Neither are required. 

So two issues to solve

- when there is a primary banner, we want show the title as the H1
- when there is *no primary banner* then we want to render the page title as an H1

Another option would be to always have the page title as H1 , and / or add a visually-hidden class to it when there is a primary banner to visually hide it. Thoughts? 

Do we mind that the page title and the h1 may be different?

**Comment by bisd** [comment]
> 

This is also true of Subsites pages #26.

**Comment by bisd** [comment]
Perhaps we should ask some content creators which option they would prefer? To me, it sounds like the second option outlined by @msayoung offers more certainty to those charged with creating content using this feature. It means they can decide whether or not to have a primary banner on design grounds without needing to worry about what is going to render as the H1. Just my two cents....

**Comment by msayoung** [comment]
@andybroomfield 
Can you give me a couple of examples of where you are using subsites at B&H ?

**Comment by msayoung** [comment]
Also to note that the primary banner is available on _all_ content types on microsites. Those pages have the page title as H1 and the banner title is just text. 

Whatever solution we put in place here will need to work there too.

**Comment by markconroy** [comment]
> Another option would be to always have the page title as H1 , and / or add a visually-hidden class to it when there is a primary banner to visually hide it. Thoughts?

I think this is my preferred option, and would follow the microsites paradigm best.

**Comment by msayoung** [comment]
Examples in the wild: 

https://www.bracknell-forest.gov.uk/libraries
https://www.lbhf.gov.uk/

**Comment by msayoung** [comment]
Adding another consideration here that if the large text isn't marked up as a heading - which it isn't by default - then it also fails WCAG 1.3.1 which says that

Information, [structure](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html#dfn-structure), and [relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html#dfn-relationships) conveyed through [presentation](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html#dfn-presentation) can be [programmatically determined](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html#dfn-programmatically-determined) or are available in text.

**Comment by msayoung** [comment]
Should we move this issue to localgov_paragraphs as it affects more than just subsites now?
